fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/common/TransformedRecordSerializerTest.java

```diff
@@ -484,17 +486,27 @@ void invalidKeyNumberEncoding() {
         assertThat(e.getMessage(), containsString("unrecognized transformation encoding"));
     }
 
-    @Test
-    void encryptRollingKeys() throws Exception {
-        RollingTestKeyManager keyManager = new RollingTestKeyManager();
+    @SuppressWarnings("UnstableApiUsage")
+    public static Stream<Arguments> randomAndCompressed() {
+        return Streams.zip(
+                RandomizedTestUtils.randomSeeds(0xC0DE6EEDL, 0x6EEDC0DEL),
+                Stream.of(Boolean.TRUE, Boolean.FALSE),
+                Arguments::arguments);
```

Comment:
This will break in nightly builds, because those run with non-0 iterations. 
I think I would use:
```
return ParameterizedTestUtils.cartesianProduct(RandomizedTestUtils.randomSeeds(), ParameterizedTestUtils.booleans("compressToo"));
```

In PRB, and when running locally, it will run with:
- `0xf17ed5eedL`, true
- `0xf17ed5eedL`, false

But in nigthly it will add two extra iterations per `compressToo` value (if I remember correctly).

I see now that `RandomizedTestUtils` does not explain this (or at least not well), I created a PR to hopefully make that clearer: https://github.com/FoundationDB/fdb-record-layer/pull/3617